find/less/__init__.py

```diff
@@ -0,0 +1,87 @@
+import check50
+import check50.c
+
+@check50.check()
+def exists():
+    """helpers.c exists."""
+    check50.exists("helpers.c")
+
+@check50.check(exists)
+def compiles():
+    """helpers.c compiles."""
+    check50.include("helpers.c")
+    check50.run("make").exit(0)
```

Comment:
There's no Makefile so this fails. The [original check](https://github.com/cs50/checks/blob/master/cs50/2017/x/find/less/check50/__init__.py) compiled these manually. This can be simplified in the new API to

```
check50.c.compile("sort.c", "helpers.c", lcs50=True)
check50.c.compile("find.c", "helpers.c", lcs50=True)
```